src/components/Calculator.js

```diff
@@ -5,36 +5,36 @@ export default class Calculator extends React.Component {
   render() {
     return (
       <div className="calculator-container">
-        <input type="text" placeholder="0"></input>
+        <input type="text" value={this.props.CalculatorScreen}></input>
         <table>
           <tr>
-            <td className="bg-light-gray">AC</td>
```

Comment:
- [ ] Your AC button is not working, please take a look at it. (It is not removing the current value from the calculator screen)